contracts/interchain-token-service/src/state.rs

```diff
@@ -27,9 +45,24 @@ pub struct ChainConfig {
     max_target_decimals: u8,
 }
 
+#[cw_serde]
+pub struct TokenBalance {
+    /// The total balance of the token bridged from/to the chain.
+    /// If the token is native to this chain, this will be the total supply bridged to other chains.
+    balance: Option<Uint256>,
```

Comment:
this optional is to indicate whether the balance should be tracked at all right? If so, I'd prefer to just not create this struct at all in that case.